tests/basics/memoryview1.py

```diff
@@ -36,3 +36,67 @@
 print(list(m[1:-1]))
 m[2] = 6
 print(a)
+
+# test slice assignment between memoryviews
+b1 = bytearray(b'1234')
+b2 = bytearray(b'5678')
+b3 = bytearray(b'5678')
+m1 = memoryview(b1)
+m2 = memoryview(b2)
+m3 = memoryview(b3)
+try:
+    m2[1:3] = m1[0:2]
+except Exception as e:
+    print(e)
+else:
+    print(b2)
```

Comment:
Oh ouch, my bad. I didn't even even remember writing the tests like that. I have to stop doing too many things at the same time :P